cmd/admin-helper/daemon.go

```diff
@@ -57,9 +57,16 @@ func daemon() error {
 }
 
 func svc() (service.Service, error) {
+	/*
+	 * The following struct fields are used to set the service's Name,
+	 * Display name, description and Arguments only when service  gets
+	 * installed via the `admin-helper install-daemon` command for CRC
+	 * in production these values are not used as the MSI installs the
+	 * service
+	 */
 	svcConfig := &service.Config{
-		Name:        "CodeReadyContainersAdminHelper",
-		DisplayName: "CodeReadyContainers Admin Helper",
+		Name:        "crcAdminHealper",
+		DisplayName: "Red Hat OpenShift Local Admin Helper",
```

Comment:
I'd prefer to keep the more generic `CRC Admin Helper` to make it usable in non-RH contexts.